.github/workflows/dockers_builder.yml

```diff
@@ -253,12 +253,6 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt update && sudo apt install -y --force-yes httpie
-      - name: Generate github private access token
-        id: ci-core-app-token
-        uses: getsentry/action-github-app-token@v2.0.0
-        with:
-          app_id: ${{ secrets.CI_CORE_APP_ID }}
-          private_key: ${{ secrets.CI_CORE_APP_PEM }}
 
       - name: Generate token for aws images
         id: app-token
```

Comment:
Maybe you can use `uses: getsentry/action-github-app-token@v2` instead of `uses: getsentry/action-github-app-token@v2.0.0` to allow for more up-to-date versions of the plugin, if available?